src/components/d3.js

```diff
@@ -609,8 +612,13 @@ export const Chart = ({
 		svg          .attr("width", width).attr("height", height).attr("viewBox", `0 0 ${width} ${height}`);
 		g            .attr("transform", "translate(" + margins.left + "," + margins.top + ")");
 
-		// Partially fix: https://github.com/Raruto/leaflet-elevation/issues/123
-		if(/Mac|iPod|iPhone|iPad/.test(navigator.platform) && /AppleWebkit/i.test(navigator.userAgent)) {
+		// Fix: https://github.com/Raruto/leaflet-elevation/issues/123
+		// Fix: https://github.com/Raruto/leaflet-elevation/issues/232
+		if (
+			/Mac|iPod|iPhone|iPad/.test(navigator.platform) &&
+			/AppleWebkit/i.test(navigator.userAgent) &&
+			!/Chrome/.test(navigator.userAgent)
+		) {
 			canvas   .style("transform", "translate(" + margins.left + "px," + margins.top + "px)");
 		}
```

Comment:
@hupe13 do you confirm that this is the patch you proposed in here? 👉 https://github.com/Raruto/leaflet-elevation/issues/232